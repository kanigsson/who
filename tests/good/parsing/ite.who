type bool
let f (b : bool) =
  {} if b then 5 else 3 {}
