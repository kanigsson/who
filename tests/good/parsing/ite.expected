type bool
let
  f  = (fun (b:bool) -> {(fun (cur:< >) -> True)} if b then 5 else 3
       {(fun (old:< >) -> (fun (cur:< >) -> (fun (anon:int) -> True)))}) 
